### PR TITLE
expose endpoint for nodle mainnet

### PR DESCRIPTION
**Pull request by ETeissonniere**
- move arcadia chain spec with the other chains
- add endpoint for nodle main

### CODE REVIEW

File: packages/apps-config/src/settings/endpoints.ts [hunk 1]
```diff
@@ -147,7 +152,7 @@ function createTest (t: TFunction): LinkOption[] {
     },
     {
       info: 'nodle',
-      text: t<string>('rpc.arcadia', 'Arcadia (Nodle Testnet, hosted by Nodle)', { ns: 'apps-config' }),
+      text: t<string>('rpc.nodle-arcadia', 'Arcadia (Nodle Testnet, hosted by Nodle)', { ns: 'apps-config' }),
```
Comment (jacogr) [inline packages/apps-config/src/settings/endpoints.ts:155]: Here you renamed the key, there has been translations done, so would just search for `rpc.arcadia` in packages/apps/public and just replace with the new.

Comment (ETeissonniere) [reply packages/apps-config/src/settings/endpoints.ts:155]: Got it!

**Review by jacogr — APPROVED** [review]
Thank you! Exciting ;)

**Comment by jacogr** [comment]
We are missing a couple types (main & test) -

image

**Pull request merged**